pkg/test/util/retry/retry.go

```diff
@@ -80,6 +80,7 @@ func UntilSuccessOrFail(t test.Failer, fn func() error, options ...Option) {
 	t.Helper()
 	err := UntilSuccess(fn, options...)
 	if err != nil {
+		time.Sleep(time.Minute * 10)
```

Comment:
remove this

Reply:
Those files were added by a bad commit.. they are removed now